src/insights_client/__init__.py

```diff
@@ -351,37 +349,62 @@ def _main():
     attempt to collect and upload with new, then current, then rpm
     if an egg fails a phase never try it again
     """
+    set_up_logging()
+    logger.debug("Verifying initial eggs to load config.")
+
+    validated_eggs = []
+    for egg in [STABLE_EGG, RPM_EGG]:
+        logger.debug("GPG verifying initial egg %s..." % (egg,))
+        if gpg_validate(egg):
+            logger.debug("GPG verification successful. Loading initial egg: %s" % (egg,))
```

Comment:
We're not loading it, it will be considered when we run the code. We're getting the information about it later as well.